### PR TITLE
minor linting and whitespace cleanup

### PR DESCRIPTION
**Pull request by H4ppy-04**
**Changes**
 - Minor linting using pyright.
 - Whitespace trimming with a nifty neovim lua command.

**Details**

Linting: pyright LSP
Whitespace/lint standard: [black](https://pypi.org/project/black/) from pypy package.

---

<div align="center">
<b>!</b> This PR is currently on hold awaiting the integration of #4 <b>!</b>
</div>

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 from configparser import ConfigParser
+import datetime
 
+from PyQt5.QtCore import QPoint, QTimer, Qt
+import PyQt5.QtGui as qtg
+import PyQt5.QtWidgets as qtw
 from win10toast import ToastNotifier
 
+
 class addTimer(qtw.QDockWidget):
```
Comment (H4ppy-04) [inline main.py]: I can add documentation strings for each of the methods in `addTimer` if you'd like @Wolfmyths.
I created a PR although I probably wouldn't rebase/merge onto base until any changes you want done pertaining to #3 have been integrated into this branch.

**Comment by Wolfmyths** [comment]
@GH-Syn Responding to your comment from PR #6 PyQt5 is cross platform if that's what you were concerned about. It's funny because I just found that out when you mentioned having a Linux distro.

I only know how to make Python programs into an `.exe` but I don't know how to make an equivalent for Linux and MacOS. (Maybe I should make a new issue about this, mobile versions might come later)

The only thing that I think is preventing Genshin Stopwatch to be cross platform is the `win10toast` module. I use it because it comes with a built-in thread parameter which makes it easy for me to implement without dealing with the quirks of PyQt5's threading.

I'll update Issue #2 about how we can make the notifications cross platform, since this comment is getting long.

**Comment by H4ppy-04** [comment]
> @GH-Syn Responding to your comment from PR #6 PyQt5 is cross platform if that's what you were concerned about. It's funny because I just found that out when you mentioned having a Linux distro.

For sure. If that's something that you'd like to be integrated I can work on making it compatible with Linux in a feature branch.

**Regarding feature branches**
Normally in repos that I create, feature branches are denoted like so:
`feat/feature-to-be-implemented` 
or if the feature is part of a scope:
`feat/scope/feature-to-be-implemented`  

Please let me know what naming convention you'd like me to follow when contributing to this repository.

**Regarding #6**
I created an initial pull request, quite impulsively, but didn't bother to use a feature branch. When I went to integrate my changes as a pull request, I noticed that another pull request had been made (#5). As to not cause merge conflicts, I closed the pull request and created a subsequent documentation branch, on which the current proposed changes live now.

I'm hesitant to replicate the changes made in #6; I auto formatted `main.py` with Black as mentioned, clashing with the proposed changes in #5. Please let me know how you'd like code to be styled and any conventions that will work best for this project.

**Comment by Wolfmyths** [comment]
**Regarding** #6 
Before merging this pull request I do want to take care of Issue #4 first and then you can make some revisions on this PR.

**Regarding Linux Support ( Now Issue #8 but Issue #2 needs to be handled first )**
I think without scope is fine. I'm new to contributing to GitHub projects and managing so sorry if I sound uncertain with these kinds of decisions.

**Comment by H4ppy-04** [comment]
> Before merging this pull request I do want to take care of Issue https://github.com/Wolfmyths/Genshin-Stopwatch/issues/4 first and then you can make some revisions on this PR.

Sure. We'll leave this one alone until #4 has been merged.

> I think without scope is fine. I'm new to contributing to GitHub projects and managing so sorry if I sound uncertain with these kinds of decisions.

No problem at all! I was in your shoes only a few months ago. If you ever want my input on something, just ask :smiley:

**Comment by Wolfmyths** [comment]
@GH-Syn I did the stylesheet rework but still needs a little polish, see #12 if interested. But now the foundation is finished we should be able to push to main after you update your branch.

**Comment by Wolfmyths** [comment]
This branch re-adds stylesheets that I've moved over to `style.py` in the rework, I combined most of the big stylesheets and put it together. Mostly the ones that start with `self.setStyleSheet`

**Comment by Wolfmyths** [comment]
It won't let me merge with the updated commit, it says:
`This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.`

And when I click on `files changed` it shows the old commit from a couple days ago.

**Comment by H4ppy-04** [comment]
how about now

**Comment by Wolfmyths** [comment]
Yeah I'm not sure why it's like this

image

**Comment by Wolfmyths** [comment]
Okay I did a double take and all that's added that doesn't need to be there is just the long stylesheets in `main.py`, I'll just merge then remove them.

**Pull request merged**